docs/userguide.md

```diff
@@ -11,11 +11,11 @@
     - [Join Clusters](#join-clusters)
     - [Checking status of joined clusters](#checking-status-of-joined-clusters)
     - [Unjoining clusters](#unjoining-clusters)
-  - [Federated API types](#federated-api-types)
-    - [Enabling federation of an API type](#enabling-federation-of-an-api-type)
+  - [Propagated API types](#propagated-api-types)
```

Comment:
Ugh, sorry.  I only meant to change it in that one heading.  The changes in this commit are not valid and need to be reverted.